adal/src/main/java/com/microsoft/aad/adal/PackageHelper.java

```diff
@@ -130,4 +130,8 @@ public static String getBrokerRedirectUrl(final String packageName, final String
         }
         return "";
     }
+
+    public AccountManager getAcctManager() {
```

Comment:
It looks like we only initialize the mAcctManager in the constructor, then you can make it final and get rid of the getter. No one is calling the getter.